tools/Performance/DynamoPerformanceTests/packages.config

```diff
@@ -1,61 +1,61 @@
+  <package id="System.Reflection" version="4.3.0" targetFramework="net48" />
+  <package id="System.Reflection.Emit" version="4.3.0" targetFramework="net48" />
+  <package id="System.Reflection.Emit.Lightweight" version="4.3.0" targetFramework="net48" />
+  <package id="System.Reflection.Metadata" version="1.6.0" targetFramework="net48" />
+  <package id="System.Resources.ResourceManager" version="4.3.0" targetFramework="net48" />
+  <package id="System.Runtime" version="4.3.0" targetFramework="net48" />
+  <package id="System.Runtime.CompilerServices.Unsafe" version="4.5.2" targetFramework="net48" />
+  <package id="System.Runtime.Extensions" version="4.3.0" targetFramework="net48" />
+  <package id="System.Runtime.InteropServices" version="4.3.0" targetFramework="net48" />
+  <package id="System.Runtime.InteropServices.RuntimeInformation" version="4.3.0" targetFramework="net48" />
+  <package id="System.Runtime.Numerics" version="4.3.0" targetFramework="net48" />
+  <package id="System.Security.AccessControl" version="4.5.0" targetFramework="net48" />
+  <package id="System.Security.Cryptography.Algorithms" version="4.3.0" targetFramework="net48" />
+  <package id="System.Security.Cryptography.Encoding" version="4.3.0" targetFramework="net48" />
+  <package id="System.Security.Cryptography.Primitives" version="4.3.0" targetFramework="net48" />
+  <package id="System.Security.Cryptography.X509Certificates" version="4.3.0" targetFramework="net48" />
+  <package id="System.Security.Principal.Windows" version="4.5.0" targetFramework="net48" />
+  <package id="System.Text.Encoding" version="4.3.0" targetFramework="net48" />
+  <package id="System.Text.Encoding.CodePages" version="4.3.0" targetFramework="net48" />
+  <package id="System.Text.Encoding.Extensions" version="4.3.0" targetFramework="net48" />
+  <package id="System.Threading" version="4.3.0" targetFramework="net48" />
+  <package id="System.Threading.Tasks" version="4.3.0" targetFramework="net48" />
+  <package id="System.Threading.Tasks.Extensions" version="4.5.2" targetFramework="net48" />
+  <package id="System.Threading.Tasks.Parallel" version="4.3.0" targetFramework="net48" />
+  <package id="System.Threading.Thread" version="4.3.0" targetFramework="net48" />
+  <package id="System.ValueTuple" version="4.5.0" targetFramework="net48" />
+  <package id="System.Xml.ReaderWriter" version="4.3.0" targetFramework="net48" />
+  <package id="System.Xml.XDocument" version="4.3.0" targetFramework="net48" />
+  <package id="System.Xml.XmlDocument" version="4.3.0" targetFramework="net48" />
+  <package id="System.Xml.XmlSerializer" version="4.3.0" targetFramework="net48" />
+  <package id="System.Xml.XPath" version="4.3.0" targetFramework="net48" />
+  <package id="System.Xml.XPath.XDocument" version="4.3.0" targetFramework="net48" />
+  <package id="System.Xml.XPath.XmlDocument" version="4.3.0" targetFramework="net48" />
```

Comment:
Do we need to have all of these `System` packages? I don't think we have anything like that in our `Dynamo.All` solution after the upgrade.

Reply:
I think these are just direct dependencies of the version of benchmarkdotnet that we use - they are not actually copied to the bin folder of the perf executable, it seems they are just referenced from the GAC at runtime.

Reply:
Oh I see. The package targets .NET Standard 2.0 only.

I see dependencies for all .NET framework specific classes it uses, but also their dependencies recursively. I guess this is expected.

Feels like we should probably consider multi-targeting for our own NuGet packages in the future, as a bloated packages.config is not nice.